### PR TITLE
Fixed Content-Length key error during image upload.

### LINKED ISSUE #921: Podman push fails on partial_update
**Version**
2.13 (https://github.com/pulp/pulp_container/commit/29b83b536da78c60d4ab0cc09d6eb58c01bce644#diff-1b944e15118f82233aa03a435e9e05c188ad0d371b71be43260506b4a5eeddffR637)

After issuing the following commands, an error is raised:
```
pulp container remote create --name lubosmj --url https://registry-1.docker.io --upstream-name lubosmj/mirror
SYNC_REPOSITORY=$(pulp container repository create --name lubosmj --remote lubosmj | jq -r ".pulp_href")
pulp container repository sync --name lubosmj

podman pull ghcr.io/pulp/test-fixture-1:manifest_d
podman tag ghcr.io/pulp/test-fixture-1:manifest_d localhost:24817/pulp/fixture:manifest_d
podman login localhost:24817 --tls-verify=false -u admin -p password
podman push localhost:24817/pulp/fixture:manifest_d --tls-verify=false
```

Traceback:
```
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/django/core/handlers/exception.py", line 47, in inner
  response = get_response(request)
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/django/core/handlers/base.py", line 181, in _get_response
  response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
  return view_func(*args, **kwargs)
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/rest_framework/viewsets.py", line 125, in view
  return self.dispatch(request, *args, **kwargs)
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/rest_framework/views.py", line 509, in dispatch
  response = self.handle_exception(exc)
File "/home/vagrant/devel/pulp_container/pulp_container/app/registry_api.py", line 253, in handle_exception
  response = super().handle_exception(exc)
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/rest_framework/views.py", line 469, in handle_exception
  self.raise_uncaught_exception(exc)
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
  raise exc
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/rest_framework/views.py", line 506, in dispatch
  response = handler(request, *args, **kwargs)
File "/home/vagrant/devel/pulp_container/pulp_container/app/registry_api.py", line 637, in partial_update
  length = int(request.headers["Content-Length"])
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/django/http/request.py", line 424, in __getitem__
  return super().__getitem__(key.replace('_', '-'))
File "/usr/local/lib/pulp/lib64/python3.10/site-packages/django/utils/datastructures.py", line 316, in __getitem__
  return self._store[key.lower()][1]
KeyError: 'content-length'
```

### PR DESCRIPTION
**Pull request by ipanova**
closes #921

### CODE REVIEW

File: CHANGES/921.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed Content-Length key error during image upload.
```
Comment (lubosmj) [inline CHANGES/921.bugfix:1]:
```suggestion
Fixed the Content-Length key error raised when uploading images.
```

**Review by lubosmj — APPROVED** [review]

**Comment by lubosmj** [comment]
We should also consider back-porting this fix.

**Review by mdellweg — APPROVED** [review]

**Comment by ipanova** [comment]
@newswangerd is 2.13 the version you need the backport?

**Comment by newswangerd** [comment]
> @newswangerd is 2.13 the version you need the backport?

@ipanova that would be perfect

**Pull request merged**